config-linux.md

```diff
@@ -686,6 +703,45 @@ The following parameters can be specified to set up seccomp:
 }
 ```
 
+### <a name="containerprocessstate" />The Container Process State
+
+The container processs state is a data structure passed via a UNIX socket.
+The container runtime MUST send the container processs state over the UNIX socket as regular payload serialized in JSON and file descriptors MUST be send using `SCM_RIGHTS`.
+The container runtime MAY use several `sendmsg(2)` calls to send the aforementioned data.
+If more than one `sendmsg(2)` is used, the file descriptors MUST be sent only in the first call.
+
+The container processs state includes the following properties:
+
+* **`ociVersion`** (string, REQUIRED) is version of the Open Container Initiative Runtime Specification with which the container processs state complies.
+* **`fds`** (array, OPTIONAL) is a string array containing the names of the file descriptors passed. The index of the name in this array corresponds to index of the file descriptor the `SCM_RIGHTS` array.
+* **`pid`** (int, REQUIRED) is the container process ID, as seen by the runtime.
+* **`metadata`** (string, OPTIONAL) opaque metadata.
+* **`state`** (map, REQUIRED) is the [state](runtime.md#state) of the container.
```

Comment:
maybe where you have `map` as the type, put the `state` as a link to that struct definition

Reply:
Good idea. Done.